test/unit/rules/no-duplicate-landmark-labels-test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const rule = require('../../../lib/rules/no-duplicate-landmark-labels');
+
+const { createErrorMessage } = rule;
+
+generateRuleTests({
+  name: 'no-duplicate-landmark-labels',
+
+  config: true,
+
+  good: [
+    '<nav aria-label="primary site navigation"></nav><nav aria-label="secondary site navigation within home page"></nav>',
+    '<nav aria-label="primary site navigation"></nav><div role="navigation" aria-label="secondary site navigation within home page"></div>',
+    '<nav aria-label={{siteNavigation}}></nav><nav aria-label={{siteNavigation}}></nav>',
```

Comment:
```suggestion
    '<nav aria-label={{siteNavigation}}></nav><nav aria-label={{siteNavigation}}></nav>',
    // since we can't confirm what the role of the div is, we have to let it pass
    '<nav aria-label="primary site navigation"></nav><div role={{role}} aria-label="secondary site navigation within home page"></div>',
```